drivers/gles3/storage/material_storage.h

```diff
@@ -316,10 +326,7 @@ struct SceneShaderData : public ShaderData {
 	bool uses_bones;
 	bool uses_weights;
 
-	uint64_t vertex_input_mask = 0;
-
-	uint64_t last_pass = 0;
-	uint32_t index = 0;
```

Comment:
I can see that they are probably unused, but are you sure?

Reply:
It wouldn't compile if they were used anywhere.